scalafmt-tests/src/test/resources/scala3/StarImport.stat

```diff
@@ -0,0 +1,17 @@
+
+<<< simple wildcard import
+import a . b . c . *
```

Comment:
btw, should we modify our import rewrite rules to change `_` to `*` for scala3?

Reply:
We could, we should probably have something like `rewrite.Scala3` setting that would change the older syntax. I will add a task to the issue.